addon/services/ember-freestyle.js

```diff
@@ -22,6 +21,23 @@ export default Ember.Service.extend({
 
   notFocused: computed.not('focus'),
 
+  ensureHljs() {
+    if (!hljsPromise) {
+      hljsPromise = new Promise((resolve) => {
+        return Ember.$.getScript('https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/highlight.min.js').done((script) => {
```

Comment:
The components render without syntax highlighting and happily wait for the `ensureHljs` promise to resolve. The highlight.js highlight call is chained off of that, so it all seems to work safely and nicely.

A log message indicating that syntax highlighting isn't available because of a timeout could be nice.